modules/osdk-bundle-deploy-olm.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-getting-started.adoc
+
+[id="osdk-bundle-deploy-olm_{context}"]
+= Bundling an Operator and deploying with Operator Lifecycle Manager
```

Comment:
I'm not sure where this is building, otherwise LGTM

Reply:
`operator-sdk run bundle` actually wasn't available yet for 4.6, so I can just remove this backported module from the PR since I couldn't use it.